src/app/_client-services/auth_client_service.ts

```
@@ -61,12 +73,12 @@ export class AuthClientService extends NextApiClientService {
 		});
 	}
 
-	static async logout(onLogout?: () => void) {
+	static async logout(isIframe?: boolean, onLogout?: () => void) {
 		this.logoutApi();
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)